addons/account/models/account_invoice.py

```diff
@@ -1045,6 +1045,7 @@ def refund(self, date_invoice=None, date=None, description=None, journal_id=None
                 'in_invoice': ('vendor bill refund')}
             message = _("This %s has been created from: <a href=# data-oe-model=account.invoice data-oe-id=%d>%s</a>") % (invoice_type[invoice.type], invoice.id, invoice.number)
             refund_invoice.message_post(body=message)
+            invoice.refund_invoice_id = refund_invoice
```

Comment:
@hmo-odoo Indeed, but there it is set on the refund invoice. Here it is set on the refunded invoice.

The use of this field is being adapted in this commit. Previously it was only used to know from which invoice the refund was made. Now it's used to get the refund from the original invoice.

I don't have an objection by adding this use for this field, it doesn't seem to break anything. The string of the field should just be adapted to match its use.